tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md

```diff
@@ -75,9 +75,11 @@ Nevertheless, there are quite a few pitfalls you can fall into when using the pl
 
 [ACCORDION-BEGIN [Step 3: ](Virtual Data Model: The new way to OData)]
 
-Now that we have explained the possible pitfalls of the current `aproach`, let's take a look at how the OData VDM of the SAP Cloud SDK simplifies the same task, as the SDK is able to incorporate more knowledge about the system that is being called.
+Now that we have explained the possible pitfalls of the current approach, let's take a look at how the OData VDM of the SAP Cloud SDK simplifies the same task, as the SDK is able to incorporate more knowledge about the system that is being called.
```

Comment:
Please add a reminder that we would need to update the approach with ODataQueryBuilder above once CXSDK is updated to SDK 3.0.0.

Reply:
Where should I add such a reminder?